src/stores/TaskStore.js

```diff
@@ -44,6 +45,47 @@ const TaskStore = types
         return null;
       }
     },
-  }));
+
+    getTextFromBbox(searchX, searchY, searchW, searchH) {
+      if (self.dataObj.ocrData) {
+        if (!self.rtree) {
+          self.createRTree();
+        }
+
+        const foundObjects = self.rtree.search({ x: searchX, y: searchY, w: searchW, h: searchH });
+
+        if (foundObjects.length > 0) {
+          const text = (foundObjects.reverse().map(obj => obj.description)).join(' ');
+
+          return text;
+        }
+      }
+      return null;
+    },
+  })).actions(self => ({
+    createRTree() {
+      if (self.dataObj.ocrData) {
+        
+        const bbrtree = RTree(10000);
+
+        const textAnnotations = self.dataObj.ocrData.outputs[0].textAnnotations;
+
+        textAnnotations.forEach((box, index) => {
+          if (index === 0)
+            return;
+              
+          bbrtree.insert({
+            x: box.boundingPoly.vertices[0].x,
```

Comment:
minor: consider extracting to a variable `const vertices = box.boundingPoly.vertices` to make the code more terse avoid duplication in the code referencing the xy coordinates.

Reply:
renamed!